src/main/webapp/user_profile.html

```diff
@@ -115,5 +49,25 @@ <h1>CREATE A NEW TASK: </h1>
             </form>
         </div>
     </div>
+    <div id="editTaskModal">
+        <div id="editModal">
+            <span id="edit-close-button" onclick="closeEditModal()">&times;</span>
+            <form id="edit-task-form" action="/tasks/edit" method="POST">
+                <h1>EDIT THE CURRENT TASK: </h1>
+                <div>
+                    <label for="edit-content-input">Task Detail:</label>
+                    <br/>
+                </div>
+                <textarea name="edit-content-input" id="edit-content-input"></textarea>
```

Comment:
"edit-detail-input"?

Reply:
Ahh I actually called "Task Detail" as "Task Content" when I first wrote the code and after changing the name to "Task Detail" I forgot to modify the name for the input textareas. I would fix it in the new commit.